packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -40,8 +40,13 @@ export class Accordion {
   @Prop({ reflect: true }) scale: Scale = "m";
 
   /**
-   * Specifies the selection mode - `"multiple"` (allow any number of open items), `"single"` (allow one open item),
-   * or `"single-persist"` (allow and require one open item).
+   * Specifies the selection mode of the component:
+   *
+   * `"multiple"`: Allows any number of selections.
+   *
+   * `"single"`: Allows only one selection.
+   *
+   * `"single-persist"`: Allows one selection and prevents de-selection.
```

Comment:
I think based on [your comment here](https://github.com/Esri/calcite-design-system/pull/8550#discussion_r1442091135), this would be modified to exclude the `:` between the property values, and each property would have commas between them?

> The two spaces at the end of each line would look identical to what you see now but here's what the other way looks like:
> ![Screenshot 2024-01-04 at 9 54 04 AM](https://private-user-images.githubusercontent.com/60022782/294259973-22af59a5-58ca-4cd4-8970-82b75556c76c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0ODIzMTQsIm5iZiI6MTcwNDQ4MjAxNCwicGF0aCI6Ii82MDAyMjc4Mi8yOTQyNTk5NzMtMjJhZjU5YTUtNThjYS00Y2Q0LTg5NzAtODJiNzU1NTZjNzZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA1VDE5MTMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5YjE4NWI4OGQ4NDM0NDAyZjdmYjQ5Y2Y1NmI1ZTgwODg3NDA4YmRiN2U4NDNjNWM5YTQ1M2VhZTM4YzRlMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.X4Y4QSJ_eybwHLaIPo-zfmKfoFva59cDfmTpFG3ez7A)
> (⬆️ I think this way is probably the one to go with IMO)

Reply:
No the first comment I made with screenshot was what it looked like before I formatted it to look the way it looks now. I figured that the format with the commas between them worked better if they weren't going to be on their own lines.

I don't have a strong opinion either way so let me know which way you think looks best or makes the most sense.

Reply:
> No the first comment I made with screenshot was what it looked like before I formatted it to look the way it looks now. I figured that the format with the commas between them looked better when the values were listed in a regular sentence format.

Gotcha, that's my mistake. I thought it was the new format proposal.

> I don't have a strong opinion either way so let me know which way you think looks best or makes the most sense.

I think the format mentioned in the picture seems the most clear, both from a repo/typings and doc site perspective - as long as you agree. So, something like:

```ts
/*
* Specifies the selection mode of the component, where:
* `"multiple"` allows any number of selected items,
* `"single"` allows only one selected item,
* `"single-persist"` allows one selected item and prevent de-selection, and
* `"none"` does not allow any selections.
*/
```

Snippets from the doc site and repo/typings projected below:

![image](https://github.com/Esri/calcite-design-system/assets/5023024/1247924e-32c3-455f-ba7e-adc92b7f69dd)

![image](https://github.com/Esri/calcite-design-system/assets/5023024/696ce20b-dff2-443e-a953-c2e6142aa211)